### PR TITLE
Documentation for Flow Designer 1.7.0, minus the release notes

### PR DESCRIPTION
**Pull request by rddheath**
What's new
1. "Get a Recommendation for Mapping Payloads for a Transformation" ( get-mapping-recommendations.adoc ), which is about how to use the Automapper feature. I don't use the term "Automapper" in the documentation, however; the term isn't used in the UI.
2. "Specify a Dynamic Value in a Text Field" ( dynamic-expression-field.adoc ), which is the first topic about using the Visual Function Editor. I don't use the term "Visual Function Editor" because the UI doesn't.
3. "Create a DataWeave Expression in a Plain Text Field" ( custom-expression-field.adoc ), which is the second and final topic about using the Visual Function Editor.
4. "Import a Mule Application" ( import-mule-app-project.adoc ) about the project-import feature.
**UPDATE on 12/13:** Badi said that the project-import feature is not going into this release. I've removed mention of it from this PR.
5. "Change the Version of Mule Runtime that a Mule App Uses" ( change-runtime-version.adoc ).

### CODE REVIEW

**Comment by rddheath** [comment]
By the way, Ryan Hollander has approved this documentation:
Ryan Hollander [20 hours ago]
@robert.heath - Looks good to me.  The notes that you added on what has changed and why you chose not to use certain terms in the customer-facing language was very helpful!  @badi.azad, I believe, is closing the loop on which pieces are included as part of this release, so I’ll defer that question.
I might have to remove mention of the automapping feature in Transform cards. I'm awaiting news from Badi.

File: modules/ROOT/pages/.get-mapping-recommendations.adoc [hunk 1]
```diff
@@ -0,0 +1,86 @@
+= Get a Recommendation for Mapping Payloads for a Transformation
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+In the Transform card, you can view recommendations for how to map the data elements in an input payload to the data elements in an output payload. If a recommendation suits your needs, you can click *Apply* to save the mapping and use it in the card.
+
+== About this task
+
+The Transform card works by converting messages from the previous card in a flow to an output messages that can be consumed by the next card. A message can consist of a payload, attributes, and variables. In this image, the Transform card is placed between and HTTP Listener card and an HTTP Request card. The Transform card can accept  messages (which contain data in JSON or in some other format) from the HTTP Listener and transform them into messages that the HTTP Request card is configured to accept.
```
Comment (valkyrie69) [inline modules/ROOT/pages/.get-mapping-recommendations.adoc]: ...from the previous card in a flow to an output messages...

Should be either flow to output messages or flow to an output message

Comment (valkyrie69) [reply modules/ROOT/pages/.get-mapping-recommendations.adoc]: In this image, the Transform card is placed between **and** HTTP Listener card

Should be "an"

File: modules/ROOT/pages/change-runtime-version.adoc [hunk 2]
```diff
@@ -0,0 +1,19 @@
+= Change the Version of Mule Runtime that a Mule App Uses
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can change an existing Mule application to use a newer version of Mule Runtime. You might want to do this if the newer version includes updates or new features that you want your application to take advantage of.
```
Comment (valkyrie69) [inline modules/ROOT/pages/change-runtime-version.adoc]: Apparently, it is now Mule runtime engine (lowercase runtime engine). Or just Mule. 

But not Mule Runtime

Comment (valkyrie69) [reply modules/ROOT/pages/change-runtime-version.adoc]: If we are using SF style guide, for versioning, they use "earlier" and "later."

File: modules/ROOT/pages/change-runtime-version.adoc [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+== Procedure
+. Open the project.
+. Click the gear icon that is in the lower-left corner of the screen.
+. In the Project Settings dialog, select the version that you want to switch to. The dialog lists a recommended version, which is the most recent version that has been tested for compatibility with Design Center.
+
+== Result
+
+Changing the version of Mule runtime might require you to update the connectors that are in the application's flows. Only after updating the connectors would be able to redeploy the application.
```
Comment (valkyrie69) [inline modules/ROOT/pages/change-runtime-version.adoc]: Mule runtime engine

This applies throughout so I won't keep commenting.

Comment (valkyrie69) [reply modules/ROOT/pages/change-runtime-version.adoc]: ? "Only after updating the connectors would be able to redeploy the application."

You can redeploy the application only after updating the connectors. 

??

File: modules/ROOT/pages/custom-expression-field.adoc [hunk 4]
```diff
@@ -0,0 +1,26 @@
+= Create a DataWeave Expression in a Plain Text Field
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+You can create DataWeave expressions in plain text fields, mixing expressions with text or using them alone, to create dynamic values when flows are triggered at runtime. A new editor provides aids for creating expressions.
```
Comment (valkyrie69) [inline modules/ROOT/pages/custom-expression-field.adoc]: JMO you can take or leave, "provides aids" doesn't sound good. Maybe tools?

File: modules/ROOT/pages/custom-expression-field.adoc [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+== About this task
+
+Every card in a flow presents a list on its right side. The first entry in the list is labelled `Expression`. (Other entries in the list appear when a card is preceded by at least one other card. However, for this task, you need only the `Expression` entry.)
+
+You can click the `Expression` entry and drop it into a text field if you want to compose a DataWeave expression within that field. If field also contains text, the result of the expression is concatenated with the text at runtime.
```
Comment (valkyrie69) [inline modules/ROOT/pages/custom-expression-field.adoc]: I think you are missing "the" or "a" in 
"If field also contains text..."

File: modules/ROOT/pages/manage-dependency-versions-design-center.adoc [hunk 6]
```diff
@@ -1,20 +1,23 @@
...
+*Result:* A list of all project dependencies appears.
 
-. Hover over a dependency, click options image:manage-dependency-versions-design-center-018a5.png[].
-. Choose Change Version > Select New Version.
-. Check that the configuration of the new version meets your needs.
+. Hover over a dependency and click the options icon: image:manage-dependency-versions-design-center-018a5.png[]
+. Select *Change Version* and select the version.
 
-== See also
+== What to do next
```
Comment (valkyrie69) [inline modules/ROOT/pages/manage-dependency-versions-design-center.adoc]: What to do Next

**Pull request merged**